frontend/src/pages/notifications/NotificationsPageView.jsx

```diff
@@ -0,0 +1,26 @@
+// import classes from './notificationspage.module.scss'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Footer from '../../components/layout/footer/FooterController'
+import Header from '../../components/layout/header/HeaderController'
+import classes from './notificationspage.module.scss'
+// import Notification from '../../components/posts/post/NotificationsPageController'
+
+const NotificationsPageView = ({ body }) => (
+    <div>
+        <Header />
+        <Box>
+            <Typography
+                className={classes.subHeader}
+                variant="h4"
+                component="div"
+            >
+                Notifications
+            </Typography>
+        </Box>
```

Comment:
The notifications text is already highlighted in the footer so it could be removed here to be more consistent with the other pages
![image](https://user-images.githubusercontent.com/54062686/158758072-469a3eba-e7f6-4719-8245-eafdef34289e.png)